src/Features/Core/Portable/ChangeSignature/DelegateInvokeMethodReferenceFinder.cs

```diff
@@ -33,29 +34,27 @@ internal class DelegateInvokeMethodReferenceFinder : AbstractReferenceFinder<IMe
         protected override bool CanFind(IMethodSymbol symbol)
             => symbol.MethodKind == MethodKind.DelegateInvoke;
 
-        protected override async Task<ImmutableArray<(ISymbol symbol, FindReferencesCascadeDirection cascadeDirection)>> DetermineCascadedSymbolsAsync(
+        protected override async Task<ImmutableArray<ISymbol>> DetermineCascadedSymbolsAsync(
```

Comment:
all the reference-finders got a *lot* simpler wrt finding 'cascaded' symbols.  In the new model, finder-cascading is just for simple (non-inheritance) based cascading (e.g. cascading from a named type to its constructors).  Inheritance cascading is handled entirely in the engine itself.